scipy/optimize/_constraints.py

```diff
@@ -249,6 +250,57 @@ def __init__(self, constraint, x0, sparse_jacobian=None,
         self.bounds = (lb, ub)
         self.keep_feasible = keep_feasible
 
+    def is_feasible(self, x):
+        """
+        The truth of whether a given vector of independent variables
+        satisfies the constraint.
+
+        Parameters
+        ----------
+        x : array-like
+            Vector of independent variables.
+
+        Returns
+        -------
+        feasible : bool
+            Whether the vector of independent variables satisfies the 'm'
+            constraints.
+        """
+        with suppress_warnings() as sup:
+            sup.filter(UserWarning)
+            ev = self.fun.fun(np.asarray(x))
+        feasible_lb = np.all(ev >= self.bounds[0])
+        feasible_ub = np.all(ev <= self.bounds[1])
+
+        return feasible_lb and feasible_ub
+
+    def excess_violation(self, x):
```

Comment:
If you don't like "excess_violation", I think just "violation" is not worse than that.

Reply:
I agree with Nikolay here...

Reply:
These comments aren't explicit, but it looks like you'd both prefer "violation".